dropwizard/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaConfigurationUtil.java

```diff
@@ -0,0 +1,512 @@
+     *
+     * @param dataSizeText the data size text, i.e. {@code 1}, {@code 1B}, {@code 1KB}, {@code 1MB},
+     *                     {@code 1GB} or {@code 1TB}
+     */
+    private static long parseDataSize(String dataSizeText) {
+        requireNonNull(dataSizeText, "text");
+        final Matcher matcher = DATA_SIZE_PATTERN.matcher(dataSizeText);
+        checkArgument(matcher.matches(),
+                      "Invalid data size text: %s (expected: %s)",
+                      dataSizeText, DATA_SIZE_PATTERN);
+
+        final long unit;
+        final String unitText = matcher.group(2);
+        if (Strings.isNullOrEmpty(unitText)) {
+            unit = 1L;
+        } else {
+            switch (Ascii.toLowerCase(unitText)) {
+                case "b":
+                    unit = 1L;
+                    break;
+                case "kb":
+                    unit = 1024L;
+                    break;
+                case "mb":
+                    unit = 1024L * 1024L;
+                    break;
+                case "gb":
+                    unit = 1024L * 1024L * 1024L;
+                    break;
+                case "tb":
+                    unit = 1024L * 1024L * 1024L * 1024L;
+                    break;
+                default:
+                    throw new IllegalArgumentException("Invalid data size text: " + dataSizeText +
+                                                       " (expected: " + DATA_SIZE_PATTERN + ')');
+            }
+        }
+        try {
+            final long amount = Long.parseLong(matcher.group(1));
+            return LongMath.checkedMultiply(amount, unit);
+        } catch (Exception e) {
+            throw new IllegalArgumentException("Invalid data size text: " + dataSizeText +
+                                               " (expected: " + DATA_SIZE_PATTERN + ')', e);
+        }
+    }
+
+    /**
+     * Resolves the given resource location to a {@link URL}.
+     *
+     * <p>Does not check whether the URL actually exists; simply returns
+     * the URL that the given location would correspond to.
+     *
+     * @param resourceLocation the resource location to resolve: either a
+     *                         "classpath:" pseudo URL, a "file:" URL, or a plain file path
+     * @return a corresponding URL object
+     * @throws FileNotFoundException if the resource cannot be resolved to a URL
+     */
+    private static URL getURL(String resourceLocation) throws FileNotFoundException {
+        // Forked from Spring 5.2.2
+        // https://github.com/spring-projects/spring-framework/blob/22a888b53df620b0905ce8beb6b0cf71981086d8/spring-core/src/main/java/org/springframework/util/ResourceUtils.java#L129
+        if (resourceLocation.startsWith("classpath:")) {
+            final String path = resourceLocation.substring("classpath:".length());
+            final URL resource = ArmeriaConfigurationUtil.class.getClassLoader().getResource(path);
+            if (resource == null) {
```

Comment:
Dropwizard has a resources class for something like this 

https://github.com/dropwizard/dropwizard/blob/master/dropwizard-util/src/main/java/io/dropwizard/util/Resources.java#L29

Reply:
That is a nice suggestion. But I do not want `Dropwizard` dependency here because of the future refactoring( https://github.com/line/armeria/pull/2373#issuecomment-571043560)